docs/api/overview.md

```diff
@@ -1,57 +1,248 @@
+
+## Sending transactions to the chain
+When sending transactions, you'll need a Vega Wallet with at least one keypair, and have the wallet service running. You'll also need to have your Vega public key (also known as a party ID) to hand, as well as the relevant transaction data, in order to submit your transaction.
+
+When your client needs to sends a transaction, it fills in the details and passes it to the wallet to be signed, and forwarded to a validator node before being added to a block.
+
+:::note Go deeper
+Read concepts about [transactions on Vega](./../concepts/vega-chain/transactions.md)
+
+You can see a full list of transaction types on the [commands API](./grpc/vega/commands/v1/commands.proto).
+:::
+
+### Transaction hashes
+Once a transaction has been successfully submitted to the chain, you receive the transaction's hash from the wallet. A transaction hash is a unique identifier for the transaction, and can be used to find that transaction and check its status on a Vega block explorer. Note that a transaction can only be seen by the block explorer once it's been processed by the network.
+
+Depending on transaction type, most will be given an ID, derived from the transaction hash, which is specific to the object the transaction creates once it's processed. You can use that object-specific ID with the relevant endpoint to then get richer, more detailed information about it. 
+
+For example, a submitted order will receive an order ID once the transaction to submit the order has been accepted into a block. You can use the REST endpoint to [get order by ID](./rest/data-v2/trading-data-service-get-order.api.mdx).
+
+## Versioning
+Once the latest code is prepared for a release, it's deployed to a testnet, where the latest features and updates can be trialled. When the validator node operators agree that a version is sufficiently stable to be released to mainnet, all code from that version and the previous versions are used to upgrade the network. 
+
+Find out what version a data node is running with the [data node info endpoint](./rest/data-v2/trading-data-service-info.api.mdx).
+
+Breaking APIs changes go through a deprecation cycle, and are announced in the summary for each [release](../releases/overview.md).
+
+The documentation on this site covers the core software version running on the Vega testnet, and the version on the Vega mainnet. Check which version's documentation you're viewing (and switch between them) by referring to the top navigation bar for each page. 
+
+See the [releases page](../releases/overview.md) for a summary of each software release and links to the full changelog on GitHub. 
+
+## REST APIs
+REST provides endpoints for querying for trading data, account information, ledger movements, asset and market information, and much more.
```

Comment:
Specify that these are coming from three different areas/IP addresses/etc. And that the recommendation is to query the data node